### ISSUE TITLE
Invalid argument supplied for foreach()

### ISSUE THREAD

**Issue by mvdhoek1** [laposta/laposta-signup-basic#5]
Invalid argument supplied for foreach() -> /laposta-signup-basic/templates/settings/settings.php:60

The following code throws an exception:
```
<?php foreach ($lists as $list): ?>
    <a class="lsb-settings__list js-list" href="#" data-list-id="<?php echo esc_attr($list['list_id']) ?>">
        <?php echo esc_html($list['name']) ?>
    </a>
<?php endforeach ?>
```

Possible solution:
```
<?php if(is_array($lists)): ?>
    <?php foreach ($lists as $list): ?>
        <a class="lsb-settings__list js-list" href="#" data-list-id="<?php echo esc_attr($list['list_id']) ?>">
            <?php echo esc_html($list['name']) ?>
        </a>
    <?php endforeach ?>
<?php endif ?>
```

**Comment by roelleor** [comment]
Thanks for submitting this issue! It should be fixed in v1.4.1.

**Comment by mvdhoek1** [comment]
That's quick, well done!
